### PR TITLE
langcode for anchoring

### PR DESCRIPTION
**Pull request by luohongyin**
v 0.1 -

w/ Oct 8 model

### CODE REVIEW

File: back-end/src/api/app_api_v1.py [hunk 1]
```diff
@@ -98,6 +99,44 @@ def get_application(app_id):
...
+    json_data = {
+        "agent_inst": create_with_inst
+    }
+    response = requests.post(
+        'https://lang-py-522564686dd7.herokuapp.com/anchoring',
+        json = json_data
+    )
+    if response.status_code != 200:
+        raise SystemError(
+            f'Failed to get valid response from server: {response.status_code}')
```
Comment (Spaceenter) [inline back-end/src/api/app_api_v1.py:117]: Better to propagate the error to API users, something like:

return Response(".......", status=500)

File: back-end/src/api/app_api_v1.py [hunk 2]
```diff
@@ -98,6 +99,44 @@ def get_application(app_id):
...
+    if app_name is None:
+        return Response("Required fields missing!", status=400)
+    app_build = DbAppBuild(id, app_name, created_by,
+                            tags, description, published, chain)
+    db.session.add(app_build)
+    
+    app_build_dict = app_build.as_dict()
+    db.session.commit()
+
+    return Response(json.dumps(app_build_dict))
```
Comment (Spaceenter) [inline back-end/src/api/app_api_v1.py:137]: return Response(json.dumps(app_build_dict), status=200)

File: back-end/src/api/app_api_v1.py [hunk 3]
```diff
@@ -98,6 +99,44 @@ def get_application(app_id):
...
+    id = gen_uuid()
+    app_name = data.get('app_name', None)
+    created_by = g.current_user_id
+    tags = data.get('tags', None)
+    description = data.get('description', None)
+    published = data.get('published', False)
+    chain = data.get('chain', None)
+
+    if app_name is None:
+        return Response("Required fields missing!", status=400)
```
Comment (Spaceenter) [inline back-end/src/api/app_api_v1.py:129]: nit: Move this check up to below line 121

**Review by Spaceenter — APPROVED** [review]

**Comment by Spaceenter** [comment]
Looks great - just some small style comments.

**Comment by luohongyin** [comment]
Thanks!

**Pull request merged**